### ISSUE TITLE
Show total count of child checkboxes in cookie?

### ISSUE THREAD

**Issue by msto** [jceb/vim-orgmode#224]
Hi,

Does vim-orgmode support the `org-checkbox-hierarchical-statistics` setting, or an equivalent?

http://orgmode.org/manual/Checkboxes.html

Thanks!

**Comment by Ron89** [comment]
Yes, add `[/]` or `[%]` after checkbox/heading. And each time you update a checkbox, the statistics will also update.

**Comment by msto** [comment]
Thanks, I had added it that, but by default it only counts the direct children. So in the examples below, I'd like the cookie next to `Project` to read `[0/4]`, which is controlled by the `org-checkbox-hierarchical-statistics` in stock org-mode.

Currently, the cookie reads `[0/0]` if its direct children don't have checkboxes, and `[0/n]` if its `n` children are checkboxed. I'd like to know if there's a setting to make it read `[0/4]` in both cases. 

Thanks!

    * Project [0/0]
      - Bugs [0/3]
        - [ ] Bug 1
        - [ ] Bug 2
        - [ ] Bug 3
      - New features [0/1]
        - [ ] Feature 1    

    * Project [0/2]
      - [ ] Bugs [0/3]
        - [ ] Bug 1
        - [ ] Bug 2
        - [ ] Bug 3
      - [ ] New features [0/1]
        - [ ] Feature 1

**Comment by Ron89** [comment]
hmm, I see your point. I'll look into it this weekend and see if it's done already, just not documented. Or if I need to implement it...

Thanks for the report.
